sdk/batch/Microsoft.Azure.Batch/src/GeneratedProtocol/Models/DataDisk.cs

```diff
@@ -57,7 +57,8 @@ public DataDisk(int lun, int diskSizeGB, CachingType? caching = default(CachingT
         /// </summary>
         /// <remarks>
         /// The lun is used to uniquely identify each data disk. If attaching
-        /// multiple disks, each should have a distinct lun.
+        /// multiple disks, each should have a distinct lun. The value must be
```

Comment:
Can we specify what "lun" stands for or means?

Reply:
This model is purely autogenerated from swagger. Generally speaking it is best to update the actual Swagger defined RestAPI documentation, because any change here will just get replaced each generation.

Reply:
Right. Can we not do it there? Or did someone else update the docs and this PR is just picking up on it?

Reply:
I guess it's the latter, since those files aren't in the change long here. Fair enough.